students/Nick_Lenssen/lesson03/mailroom.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+from sys import exit
+
+
+def create_report():
+    print("\n{:<22}{:<5}{:<20}{}{:<25}{}{:<15}".format(*('Donor Name','|','Total Given','|','Num Gifts','|','Average Gift')))
+    print ('-'*90)
+    for val, i in enumerate(main_list[0]):
+        print ("{:<20} {:>9} {:>9} {:>26}{:>11}{:>12}".format(*(i, '$', sum(main_list[val+1]), len(main_list[val+1]), '$',round(sum(main_list[val+1])/len(main_list[val+1]),1))))
+        
+def use_it(full_name):
+    while True:
+        amount = input("\nPlease enter in the amount you want to donate: ")
+        if amount == 'c':
+            main_list[0].pop()
+            main_list.pop()
+            print (main_list)
+            main()
+        else:
+            amount = int(amount)
+            main_list[main_list[0].index(full_name)+1].append(amount)
+            print ('Thank you {} for your generous donation of {}'.format(full_name, amount))
+            main()
+def send_thanks():
+    full_name = input("\n\nPlease enter a name for a Thank You to go out: ")
+    while True:
+        while full_name == 'list':
+            print (main_list[0])
+            full_name = input("\n\nPlease enter a name for a Thank You to go out: ")
+        if full_name == 'c':
```

Comment:
is 'c' a quit option? should the user know about this option?

add `return` statement don't call main() directly